.circleci/config.yml

```diff
@@ -543,7 +543,10 @@ jobs:
           command: |
             cd crypto/crypto && \
             RUST_BACKTRACE=1 $CARGO $CARGOFLAGS test \
-              --features='vanilla' \
+              --features='vanilla-u64' \
+              --no-default-features && \
+            RUST_BACKTRACE=1 $CARGO $CARGOFLAGS test \
```

Comment:
I'm afraid you'll need `no-default-features` here as well, otherwise you'll hit the exclusivity check on fiat + vanilla-u32.

Reply:
`no-default-features` is included on last line of both test clauses.

LMK if I am missing something obvious here.

Reply:
:facepalm: